docs/lara-plugin-api/README.md

```diff
@@ -258,12 +252,6 @@ ___
 
 *Defined in [popup.ts:86](../../lara-typescript/src/plugin-api/popup.ts#L86)*
 
-Ask LARA to add a new popup window.
-
-Note that many options closely resemble jQuery UI dialog options which is used under the hood. You can refer to jQuery UI API docs in many cases: [https://api.jqueryui.com/dialog](https://api.jqueryui.com/dialog) Only `content` is required. Other options have reasonable default values (subject to change, so if you expect particular behaviour, provide necessary options explicitly).
-
-React warning: if you use React to render content, remember to call `ReactDOM.unmountComponentAtNode(content)` in `onRemove` handler.
-
```

Comment:
Right, I didn't notice it. I tried to run docs generation again and it's still the same. Maybe it's related to the recent dependencies update (e.g. TypeScript 3.9)? @dougmartin, does doc generation work for you? Doesn't it remove typedocs comments like here?

When I updated Typedoc and markdown plugin, it actually helped. But changed a bit structure of generated docs. So, not sure I should commit it here.

Also, a related question - when I was working on plugin-api, there's specific setup of files and directories, so the generated docs are showing **only** members exported in NPM package. In case of LARA Interactive API, it seems that docs are generated for everything, even the internal helpers. Do we want to work on that, so these files are not included in the docs? If so, it should be separate PR I think. But wondering if it makes sense, as probably most devs won't read that anyway (but use typing in IDE).

Reply:
I think improving the documentation for the interactive api should be its own story and PR. We can deal with the mixing of internal types with external ones.  I do think it is important to do since I think it would be good to have an automatically generated reference which can be linked to.

Moving the fix for the lara plugin api docs to a separate PR seems also fine to me.

Reply:
I added a story for this documentation fix: https://www.pivotaltracker.com/story/show/173116252